src/main/java/marquez/db/mappers/JobRowMapper.java

```diff
@@ -40,6 +40,7 @@ public Job map(@NonNull ResultSet results, @NonNull StatementContext context)
         arrayOrThrow(results, Columns.INPUT_DATASET_URNS),
         arrayOrThrow(results, Columns.OUTPUT_DATASET_URNS),
         timestampOrThrow(results, Columns.CREATED_AT),
-        timestampOrThrow(results, Columns.UPDATED_AT));
+        timestampOrThrow(results, Columns.UPDATED_AT),
+        Job.Type.valueOf(results.getString(Columns.JOB_TYPE)));
```

Comment:
I would use `Columns.stringOrThrow()` to access the job type column.

Reply:
Yea agreed - fixed.